tests/crd-controller/FlowTest.cs

```diff
@@ -1,23 +1,31 @@
 using System;
-using System.Diagnostics;
-using System.IO;
-using System.Reactive.Linq;
-using System.Reactive.Subjects;
-using System.Text;
-using System.Threading.Tasks;
-using k8s;
-using k8s.Models;
+using System.Diagnostics;
+using System.IO;
+using System.Reactive.Linq;
+using System.Reactive.Subjects;
+using System.Text;
+using System.Threading.Tasks;
+using k8s;
+using k8s.Models;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace crd_controller
 {
     public class FlowTest
     {
+        private readonly ITestOutputHelper mTestOutputHelper;
+
+        public FlowTest(ITestOutputHelper testOutputHelper)
+        {
+            mTestOutputHelper = testOutputHelper;
+        }
+
         [Fact]
-        public async Task Create_Delete_KamusSecret()
+        public async Task Create_Update_Delete_KamusSecret()

```

Comment:
Maybe add different flow for an update? So we can know easily what's not working. E.g. create a secret and KamusSecret without testing it and then apply the update.